package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.3.5",
```

Comment:
In readme, I can see that your MIT license is broken kindly fix it up as well. Thank you!